grouper/audit.py

```diff
@@ -130,3 +133,15 @@ def get_audits(session, only_open):
         query = query.filter(Audit.complete == False)
 
     return query
+
+
+def get_auditors_group(session):
+    group_name = get_auditors_group_name(settings)
+    if not group_name:
+        raise NoSuchGroup('Please ask your admin to configure the default auditors group name')
+    auditors_group = Group.get(session, name=group_name)
+    if not auditors_group:
+        raise NoSuchGroup('Please ask your admin to configure the default group for auditors')
+    # maybe we can insist that the auditors group actually has the
```

Comment:
we probably *should* crash here or the background processor if the auditors group doesn't actually have to audit permission. this is so that if the group is misconfigured, the background processor crashes so we know to take a look; otherwise it just keeps adding non-auditor approvers to the group but that effectively means nothing because the next time around, the users will again have no audit permission.

essentially this would be failing silently. or, instead of crashing, we send an email to grouper admin. but i think crashing is the easy solution and should only happen once or very rarely, and so the disruption should be minimal. and also emailing doesn't guarantee that emails will be read :)

Reply:
Yea, logic makes sense to me. Maybe on principle add a quick comment on the function saying that it'll crash? Or to be really aggressive add a flag to control behavior but don't think that's necessary at this point

Reply:
have made this function do the check and raise exception if the permission is not in the group